src/font.rs

```diff
@@ -119,3 +120,12 @@ pub fn width_from_chars(len: usize, config: &config::Font) -> f32 {
     .expand(Size::new(1.0, 0.0))
     .width
 }
+
+pub fn get(font_style: Option<FontStyle>) -> Option<Font> {
+    font_style.and_then(|font_style| match font_style {
+        FontStyle::Normal => None,
+        FontStyle::Bold => Some(MONO_BOLD.clone()),
+        FontStyle::Italic => Some(MONO_ITALICS.clone()),
+        FontStyle::ItalicBold => Some(MONO_BOLD_ITALICS.clone()),
+    })
+}
```

Comment:
We're making assumptions about how the caller should call this by taking an `Option<_>` instead of just a `FontStyle`. Also, we can remove the return `Option` by mapping `Normal -> MONO`. I think this makes the function much more "functional" since it can be easily used w/ `Option` methods like `map` if the caller has an `Option<FontStyle>`